README.md

```diff
@@ -21,13 +21,18 @@ by Google on GitHub.
 | [sync-repo-settings] | Synchronize repository settings from a centralized config | [install][sync-repo-settings-app] |
 | [trusted-contribution] | Allows Kokoro CI to trigger for trusted contributors | [install][trusted-contribution-app] |
 
-## Testing Locally
+## Development environment
+
+You need to install node.js version 12 or higher.
```

Comment:
nit: I might be worth directing people to `nvm` here; I find I'm usually testing against a few Node.js versions at once.

Reply:
Done